### DISCUSSION TITLE
Creating multiple db databases reporting errors

### DISCUSSION THREAD

**Discussion by xuyongweb** [ujjwalguptaofficial/JsStore#322] [Q&A]
Creating multiple db databases reporting errors
Before each creation, the connection was closed and the database name changed, but it still reported that it already exists. Just refresh it

`let connection = null
        connection = new Connection();
        connection.on("open",(db)=>{
            console.log("connection opened", db);
        });
        connection.on("upgrade",(database, oldVersion, newVersion, )=>{
            console.log(`database is upgraded from ${oldVersion} to ${newVersion} and database looks like`, database);
        });
        connection.addPlugin(workerInjector);
        console.log(connection)
        const dataBase = {
          name: `hoxin_` + this.id,
          tables: initTable,
          version: 1
        };
        await connection.initDb(dataBase)
        await connection.terminate()
        this.id ++ `

**Comment by xuyongweb** [comment]
`
let connection = null
        connection = new Connection();
        connection.on("open",(db)=>{
            console.log("connection opened", db);
        });
        connection.on("upgrade",(database, oldVersion, newVersion, )=>{
            console.log(`database is upgraded from ${oldVersion} to ${newVersion} and database looks like`, database);
        });
        connection.addPlugin(workerInjector);
        console.log(connection)
        const dataBase = {
          name: `hoxin_` + this.id,
          tables: initTable,
          version: 1
        };
        await connection.initDb(dataBase)
        await connection.terminate()
        this.id ++ 
`

**Comment by xuyongweb** [comment]
Create a database by segregating messages and logging in to different people, but if you switch logins and log in again, the creation will fail
Version change transaction was aborted in upgradeneeded event handler

**Reply by ujjwalguptaofficial** [reply]
What you mean is terminating connection and reintializing again is creating the issue ?
